chartpress.py

```diff
@@ -566,20 +567,38 @@ def publish_pages(chart_name, chart_version, chart_repo_github_path, chart_repo_
     this, it is as we would have a --force-publish-chart by default.
     """
 
-    # clone the Helm chart repo and checkout its gh-pages branch,
-    # note the use of cwd (current working directory)
-    checkout_dir = '{}-{}'.format(chart_name, chart_version)
-    check_call(
-        [
-            'git', 'clone', '--no-checkout',
-            git_remote(chart_repo_github_path),
-            checkout_dir,
-        ],
-        # warning: if echoed, this call could reveal the github token
-        echo=True,
-    )
+    # clone/fetch the Helm chart repo and checkout its gh-pages branch, note the
+    # use of cwd (current working directory)
+    checkout_dir = f'{chart_name}-{chart_version}'
```

Comment:
With all the changes from `some_string.format()` to f-strings: does chartpress specify a minimum Python version it requires? If yes we need to bump that up if it is too old.

Reply:
Yepp I checked this, its py36 already which is part of the test suite!